host/contracts/contracts.go

```diff
@@ -115,16 +115,17 @@ type (
 
 		// NegotiationHeight is the height the contract was negotiated at.
 		NegotiationHeight uint64 `json:"negotiationHeight"`
-		// RevisionConfirmed is true if the contract revision transaction has
-		// been confirmed on the blockchain.
-		RevisionConfirmed bool `json:"revisionConfirmed"`
-		// FormationConfirmed is true if the contract formation transaction
-		// has been confirmed on the blockchain.
+		// FormationIndex is the height at which the contract has been confirmed
+		// on the blockchain. If the contract has not been confirmed, the field
+		// is the zero value.
 		FormationIndex types.ChainIndex `json:"formationIndex"`
+		// RevisionConfirmed is true if the latest revision has been confirmed
+		// on the blockchain.
+		RevisionConfirmed bool `json:"revisionConfirmed"`
 		// ResolutionIndex is the height the resolution was confirmed
 		// at. If the contract has not been resolved, the field is the zero
 		// value.
-		ResolutionIndex types.ChainIndex `json:"resolutionHeight"`
+		ResolutionIndex types.ChainIndex `json:"resolutionIndex"`
```

Comment:
I'd say that usually this is illegal but from what I can tell this endpoint is new and not documented so maybe fine to fix it here?